src/js/components/Home/Countries.js

```diff
@@ -1,16 +1,21 @@
 import React, {Component} from "react";
 import Map from "./Map";
 import ReactFlagsSelect from 'react-flags-select';
-import  countries from 'react-flags-select/es/countries';
+import countries from 'react-flags-select/es/countries';
 import Geocode from "react-geocode";
+import Book from "../../../Assests/Images/book.png";
+import {NavLink} from "react-router-dom";
+import Speedy from '../../../Assests/Images/Speedy_Pose_3.png'
+
 
 Geocode.setApiKey("AIzaSyBiCz9njYP8nYxEUEQLzltCM8kAnm7z6To");
```

Comment:
I know this isn't from your PR but should these keys not be in environment variables

Reply:
we are looking into a S3 AWS bucket and we want to add all our files/ contents to the bucket. I just included it locally to get the wireframe up and as close as possible to show the client when needed. Once the S3 bucket is configured the content will be moved and our DB will be reconfigured to allow the change.